src/libmongoc/doc/includes/init_cleanup.txt

```diff
@@ -1,3 +1,5 @@
 Initialize the MongoDB C Driver by calling :symbol:`mongoc_init` exactly once at the beginning of your program. It is responsible for initializing global state such as process counters, SSL, and threading primitives.
 
+Exception to this is `mongoc_log_set_handler`, which should be called before `mongoc_init()` or some log traces would not use your log handling function. See logging document for details.
```

Comment:
> See logging document for details

I'm not sure how to link documents in .rst files. Could you provide an example, please?

Reply:
Sure. See the links in the webpage for aggregate opts [here](http://mongoc.org/libmongoc/current/mongoc_collection_aggregate.html), specifically `collation`. The text file for it is in aggregate-opts.txt in this same directory.

Linking the logging header would be similar to the header link for `Setting Collation Order`. In other words,
1) create an anchor to the header in src/libmongoc/doc/logging.rst (since one doesn't exist). For example,
```
.. _custom_log_handlers: // <- add anchor

  Custom Log Handlers
 -------------------

``` 
2) Now you can link to that header from this text file. For example,
```
 :ref:`Custom Log Handler<custom_log_handlers>`
```

If you'd like to test this locally you can follow the steps [here](http://mongoc.org/libmongoc/1.17.0/installing.html#generating-the-documentation) for building the documentation. Afterwards, the HTML files will be in /build/src/libmongoc/doc/html.